tests/acceptance/account_details-test.js

```diff
@@ -0,0 +1,86 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import {make} from 'ember-data-factory-guy';
+import { mockFindAll } from 'ember-data-factory-guy';
+
+var App, user, organisation, organisationsUser, gogo_van, order, pkg, ordersPackage, gcOrganisations;
+
+module('Acceptance | Account Details Page', {
+  beforeEach: function() {
+    window.localStorage.authToken = '"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjo2LCJpYXQiOjE1MTg3NzI4MjcsImlzcyI6Ikdvb2RDaXR5SEsiLCJleHAiOjE1MTk5ODI0Mjd9.WdsVvss9khm81WNScV5r6DiIwo8CQfHM1c4ON2IACes"';
```

Comment:
Why do we need authToken here  when `startApp` function used below already lets you log in?